service-onboarding-build-pack/Jenkinsfile

```diff
@@ -369,19 +436,22 @@ node  {
 					echo error
 				}
 
-				sendEvent(service_type, service_name, domain, runtime, admin_group, Event_Name.ADD_WEBHOOK, Event_Status.STARTED, "")
-				try{
-					echo "Adding web hooks for Jenkins integration"
-					sh "curl -X PUT -k -v -u \"$UNAME:$PWD\" -H \"Content-Type: application/json\"  " + var_bitbucket_webhook_api + bitbucket_name + "/configurations  -d \'{\"title\": \"notify-jenkins\", \"url\": \"" + var_scm_source_hook + "\" , \"enabled\": true}\'"
-					sendEvent(service_type, service_name, domain, runtime, admin_group, Event_Name.ADD_WEBHOOK, Event_Status.COMPLETED, "")
-				}
-				catch(error){
-					updateServiceInDB(service_id_in_db, env.API_KEY, region, "creation_failed", repo_url);
+				if(scm != "gitlab"){
```

Comment:
same as above, move to a function

Reply:
The number of parameters that needs to be send in this case is more than 12+ , so I thin it would be better it stay rather than make it a method.